.github/workflows/coverity.yml

```diff
@@ -156,44 +156,55 @@ jobs:
         run: |
           rm -rf ${COVERITY_TOOL_DIR} && mkdir -p ${COVERITY_TOOL_DIR}
           pushd ${COVERITY_TOOL_DIR}
-            wget https://scan.coverity.com/download/linux64 --progress=bar:force:noscroll --post-data "token=${{ secrets.COVERITY_TOKEN }}&project=openvino" -O coverity_tool.tgz
-            tar -I pigz -xf coverity_tool.tgz && rm coverity_tool.tgz
+            wget https://ubit-artifactory-or.intel.com/artifactory/coverity-or-local/Enterprise/license.dat -O license.dat
```

Comment:
Installation procedure from: https://intel.sharepoint.com/:p:/r/sites/SecureToolsSoftwareCoP/_layouts/15/Doc.aspx?sourcedoc=%7B79857C49-DDF6-4B19-9D47-F2164487ADB5%7D&file=Coverity_Basics_Training.pptx&action=edit&mobileredirect=true
This instruction I used in my experiments, not sure if simple copy will work in the same way (possible copy source: https://ubit-artifactory-or.intel.com/artifactory/coverity-or-local/Enterprise/cov-analysis-linux64-2024.6.1.tar.gz - see: https://wiki.ith.intel.com/display/SecTools/Coverity+Downloads)